src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UserPrefs.java

```diff
@@ -100,6 +100,7 @@ public void writeUserPrefs()
 
    public void writeUserPrefs(CommandWithArg<Boolean> onCompleted)
    {
+      UpdatePrefs(session_.getSessionInfo().getPrefs());
```

Comment:
nit: method name should be camelCase?

Reply:
(realize this is a pre-existing method but maybe worth changing while you're working with this code)